.buildkite/bk.integration.pipeline.yml

```diff
@@ -335,15 +356,15 @@ steps:
         matrix:
           setup:
             variants:
-            - "basic,slim,complete,service,elastic-otel-collector"
-            - "wolfi,slim-wolfi,complete-wolfi,elastic-otel-collector-wolfi"
+              - "basic,slim,complete,service,elastic-otel-collector"
+              - "wolfi,slim-wolfi,complete-wolfi,elastic-otel-collector-wolfi"
             version:
-            - v1.27.16
-            - v1.28.9
-            - v1.29.8
-            - v1.30.8
-            - v1.31.0
-            - v1.32.0
+              - v1.27.16
```

Comment:
let's avoid these kind of changes, they are unrelated to the nature of the PR.

Reply:
Fixed